### PR TITLE
Client: A Checkbox to switch which button confirms things.

### PR DESCRIPTION
**Pull request by Xkower8181**
A bit a small pet peeve of mine, which was a good thing to code in order to familiarize myself with the codebase more.
The only problem being is that the code (perhaps) is ugly. That's it.

### CODE REVIEW

File: Client/gamemanager.cpp [hunk 1]
```diff
@@ -153,6 +153,7 @@ GameWidget* GameManager::createGame(ppvs::GameSettings* gs, const QString& roomN
 	gs->controls["1left"] = settings.string("controlsp1", "left", "left").toStdString();
 	gs->controls["1right"] = settings.string("controlsp1", "right", "right").toStdString();
 	gs->controls["1start"] = settings.string("controlsp1", "start", "return").toStdString();
+    gs->SwapABConfirm = settings.boolean("controlsp1", "swapabconfirm", true);
```
Comment (jchv) [inline Client/gamemanager.cpp:156]: It looks like this is set to default to `true`. Presumably it should default to `false`.

**Review by jchv — CHANGES_REQUESTED** [review]
Mostly LGTM, but it looks like all of the code changes are using spaces instead of tabs. Up until very recently the code was inconsistent, but I've moved it to all use tabs. If possible, it would be most preferable to setup ClangFormat and run all files you're editing through it prior to merging. It will take care of this and other issues.

For Qt Creator, it needs to be enabled manually:

https://www.qt.io/blog/2019/04/17/clangformat-plugin-qt-creator-4-9

**Comment by jchv** [comment]
(It would be nicer if internally we swapped a designated 'confirm' button instead of having it handled at each place where a confirmation is made, but it's not trivial to do that right now, so that can wait until later.)

**Comment by Xkower8181** [comment]
I've done everything that you have mentioned plus an oversight that you missed in settingsdialog.ui where it automatically opened into the controls tab instead of general

**Review by jchv — APPROVED** [review]
OK, I ran the formatter because it was still weird, but otherwise everything is good to merge I think.

**Pull request merged**